.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu", "macos"]
-        python-version: ["3.9", "3.10", "3.11"]
+        python-version: ["3.10", "3.11"]
```

Comment:
Why not adding 3.12 here?

Reply:
I didn't think it was good to add 3.12 to our workflows when we would still get an installation failure because of the unavailability of tensorflow. When that is solved I agree we should add it.